test/app-tap/datetime.test.lua

```diff
@@ -1830,6 +1830,565 @@ test:test("Parse strptime format", function(test)
+            fmt = '%b',
+        },
+        {
+            buf = 'Apr',
+            fmt = '%b',
+        },
+        {
+            buf = 'May',
+            fmt = '%b',
+        },
+        {
+            buf = 'Jun',
+            fmt = '%b',
+        },
+        {
+            buf = 'Jul',
+            fmt = '%b',
+        },
+        {
+            buf = 'Aug',
+            fmt = '%b',
+        },
+        {
+            buf = 'Sep',
+            fmt = '%b',
+        },
+        {
+            buf = 'Oct',
+            fmt = '%b',
+        },
+        {
+            buf = 'Nov',
+            fmt = '%b',
+        },
+        {
+            buf = 'Dec',
+            fmt = '%b',
+        },
+        -- %B
+        {
+            buf = 'January',
+            fmt = '%B',
+        },
+        {
+            buf = 'February',
+            fmt = '%B',
+        },
+        {
+            buf = 'March',
+            fmt = '%B',
+        },
+        {
+            buf = 'April',
+            fmt = '%B',
+        },
+        {
+            buf = 'May',
+            fmt = '%B',
+        },
+        {
+            buf = 'June',
+            fmt = '%B',
+        },
+        {
+            buf = 'July',
+            fmt = '%B',
+        },
+        {
+            buf = 'August',
+            fmt = '%B',
+        },
+        {
+            buf = 'September',
+            fmt = '%B',
+        },
+        {
+            buf = 'October',
+            fmt = '%B',
+        },
+        {
+            buf = 'November',
+            fmt = '%B',
+        },
+        {
+            buf = 'December',
+            fmt = '%B',
+        },
+        -- %c
+        {
+            buf = 'Thu Jan  1 03:00:00 1970',
+            fmt = '%c',
+        },
+        -- %C
+        {
+            buf = '26-08-2024 20',
+            fmt = '%d-%m-%Y %C',
+        },
+        -- %d
+        {
+            buf = '01',
+            fmt = '%d',
+        },
+        {
+            buf = '31',
+            fmt = '%d',
+        },
+        -- %D
+        {
+            buf = '09/09/04',
+            fmt = '%D',
+        },
+        -- %e
+        {
+            buf = ' 30',
+            fmt = '%e',
+        },
+        -- %E, modifier, skipped.
+        -- %F
+        {
+            buf = '1970-01-01T03:00:00.125',
+            fmt = '%FT%T.%f',
+        },
+        {
+            buf = '1970-01-01T03:00:00.125 MSK',
+            fmt = '%FT%T.%f %Z',
+        },
+        {
+            buf = '1970-01-01T03:00:00.125+0300',
+            fmt = '%FT%T.%f%z',
+        },
+        {
+            buf = '1970-01-01',
+            fmt = '%F',
+        },
+        -- %f, Tarantool-specific.
+        {
+            buf = '100',
+            fmt = '%f',
+        },
+        -- %G
+        {
+            buf = '2024',
+            fmt = '%G',
+        },
+        -- %g, it is not possible to roundtrip, skipped.
+        -- %h, same as %b
+        {
+            buf = 'Jan',
+            fmt = '%b',
```

Comment:
Looks like you forgot to update to the `%h` here and below.

Reply:
Thanks!

Fixed:

```diff
         -- %h, same as %b
         {
             buf = 'Jan',
-            fmt = '%b',
+            fmt = '%h',
         },
         {
             buf = 'Feb',
-            fmt = '%b',
+            fmt = '%h',
         },
         {
             buf = 'Mar',
-            fmt = '%b',
+            fmt = '%h',
         },
         {
             buf = 'Apr',
-            fmt = '%b',
+            fmt = '%h',
         },
         {
             buf = 'May',
-            fmt = '%b',
+            fmt = '%h',
         },
         {
             buf = 'Jun',
-            fmt = '%b',
+            fmt = '%h',
         },
         {
             buf = 'Jul',
-            fmt = '%b',
+            fmt = '%h',
         },
         {
             buf = 'Aug',
-            fmt = '%b',
+            fmt = '%h',
         },
         {
             buf = 'Sep',
-            fmt = '%b',
+            fmt = '%h',
         },
         {
             buf = 'Oct',
-            fmt = '%b',
+            fmt = '%h',
         },
         {
             buf = 'Nov',
-            fmt = '%b',
+            fmt = '%h',
         },
         {
             buf = 'Dec',
-            fmt = '%b',
+            fmt = '%h',
         },
```